manifests/plugin/remote_execution/ssh.pp

```diff
@@ -63,4 +63,17 @@
       creates => $ssh_identity_path,
     }
   }
+
+  case $::osfamily {
+    'Debian': {
+      $core_package = 'ruby-smart-proxy-remote-execution-ssh-core'
+    }
+    default: {
+      $core_package = 'tfm-rubygem-smart_proxy_remote_execution_ssh_core'
+    }
+  }
+
+  package { $core_package:
+    ensure => installed',
+  }
```

Comment:
Where would you create the variable? And where would it be used?

We need more than just the plugin prefix, we need the _ to - for debian, rubygem- prefix, etc.